### DISCUSSION TITLE
Website Feedback Devlog 12/13/2023

### DISCUSSION THREAD

**Discussion by TehTurk** [Yellow-Dog-Man/Resonite-Issues#899] [General]
I really enjoyed the presentations and concepts they had a really organic feel to it, and positively presents the platform. I just had a few issues if that's the full direction going forward ( I saw request for feedback so hello 👋🏻 ), or if I could at least input, and welcome others to chime in on what's the best way to go about it for featuring creators on the website.

I like the concept of having handpicked creators to feature on the website, but from experience I feel like it'd cause some issues? (Who gets picked over others, people being catty, etc etc)

Would it be more easier to recommend a Creator/Creation of the Month based on metrics of some sort and avoiding repeat creators? Or alternatively, allowed folks to make a creator profile of some sort, link to their creations, and then have them linked if folks see their creations featured on a blog post or page. (Because one hand, are you making a yellow-book for creators, or are you trying to ease the access or knowledge of these notable creators by doing this. Featuring em is one thing, having them be constantly at reach is also another)

There's also other aspects, where when you set specific areas of interest for folks that are highlight on a page as such, it also limits what focus may focus on creating within the community too. (Gadgets, Live Perf, Avatar Makers etc)

I'm ok with us highlight notable, or noteworthy creators as you noted being hands on with creators being represented but I'm also wary of it becoming too echo chambery with how things develop over time, because there are issues with constant spotlights vs momentary spotlights and people slot-ing themselves heavily in a slice to get recognized as well.
![Discord_Spc8pMUV1k](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/26357446/5f378ee5-a36f-4ae5-8e08-99efc54b20ef)

![Discord_vasSl8MBxy](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/26357446/f723121d-c2a6-4c09-be12-3305259986c7)

**Comment by brodokk** [comment]
For the spotlight categories i'm wondering if an "external tool creators" or something like that could also be interesting?

Like mods creators, headless tool creators or even proxy creators (this one would probably disappear when their will be json support)

**Reply by ChromaShiba** [reply]
As a category I think this is a great idea!

**Comment by brodokk** [comment]
While @TehTurk is talking about the potential limit of a system like that I'm wondering if having groups that are just doing social events could be also an interesting category too. This more centered of creations and educations.

**Reply by TehTurk** [reply]
I'd love this too tbh, us organizer folks don't get enough credit 😤, but that could also just be events in general.

**Comment by anquietas-sys** [comment]
Perhaps if there was a sort of form you could fill out on the website where you can put in info about yourself and your creations, and you could make changes to it later down the line as well
Additionally, if you could click on a creator's tab and get a more in depth window with more images or information, that would be interesting

**Reply by ChromaShiba** [reply]
For sure an avenue we could go down for this.

**Comment by ChromaShiba** [comment]
To Turks point, the running idea is that there would be an open call to the community to submit to these listings (with some moderation that would just make sure the type of content is in line with our policies).

There is definitely some concern from myself about how to approach this as we would not like to appear favorable to certain types of content over others. Having specific creator accounts linked to these is an interesting thought, it was also suggested outside this thread that creators could stand up their own Wix style pages to link to from this directory. 

I am appreciative of the discussion here so far! It's entirely my intention to be talking about this idea early before shipping something to make sure we do good by our creators here.

**Reply by TehTurk** [reply]
I like the idea of the open call, but feel like it'd cause some issues? (I'm just usually haphazard with those words lol) Because if I make a comparison it'll eventually kind of turn into an Art website ala ArtStation, deviantart, or pixiv to name a few. So I imagine similar precedents or considerations could be pulled from that. This is why I suggested focusing on some Creator of the month, or group, item, tool etc and let folks be linked that way (linking to a page, in which folks can check out other stuff). 

Because it's also like a different medium in which one is presenting their creations too, so there's dolling it up proper, thumbnails, etc. For some it'd eventually end up becoming a skill set people would learn to generate clicks. I'm open to it and up for ideas, just not sure what's the best direction/idea.

**Reply by ChromaShiba** [reply]
This is another way to potentially approach it, and possibly better to direct to a community run directory (directories?) of creators. I personally have enjoyed art websites as a resource as a creative. If we could properly automate a forum like listing, that could ease some of the headache that comes with curation. I'd personally want to give as much power to the creators to showcase themselves in the way they most prefer.    

As a resource, it feels like there is a nice balance to be had between featuring content and being able to discover creators. Maybe the current mockup (as posted at the top of this discussion) is just another page to add to the utility here (upon adding the ability for creators to self tag themselves into a category of work). 

Just spitballing some thoughts.

**Reply by TehTurk** [reply]
Totally agree, I could see the categories even being user generated ala steam tags in some form as well. I also don't know if any marketplace talks have been had, but it'd also likely help in the future with profiles if folks want to sell creations too.

**Comment by ChromaShiba** [comment]
<img width="1505" alt="Screenshot 2023-12-14 135943" src="https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/79342865/c9b10fae-8ff3-4e1e-b4c1-cee069c0d4e0">

Quickly added modding/external tooling to this concept

**Comment by GearBell** [comment]
While I am not opposed to the concept, I think it is a very bad idea to put it on the official website for several reasons. I was hesitant to give my opinion but was gently coaxed into giving my voice. My reasoning will make people upset but it is blunt and to the point. Brand risk. Again I am not opposed to this concept, but am very highly opposed to putting it on the official website. The website should be about the platform itself. There is nothing wrong with advertising the community or players on other advertisement platforms such as twitch and twitter and such. But for Resonite.com, it should just be about the platform itself. Here are a few reasons why I am *very* concerned if this concept is put on the official website:
1) Ego - I have witnessed that many in the community loves hanging "special treatment" stickers over others and it causes unfortunate ego flaring. I have seen people scramble and boast about badges and stickers and become insufferable about it. (And before people attack me with "being a hypocrite because youre on there" I was asked to be photographed, it was their choice, and have not brought it to anyones attention since then.)
2) Website upkeep - It will be a constant thing for updating. If it was its own side website, like a community one (I dunno, "ResoniteCommunity.com"?), then it frees up the main website to advertise *the program itself* first, then a casual hyperlink to go to the community website for its own updating.
3) Is it a profile or profile-to-gallery? - Harsh restrictions will have to be in place because often people put superficial poems to express their purposes or resort to using "cutesy" language that will inevitably weird out newcomers/companies going to the official website to look it up. If its a profile-to-gallery there *must* be a number limitation to how many pictures they can upload. That *will* be spammed. If it is limited to 10 images to upload then moderation can more easily check and approve.
4) Presentability - Some people have lost touch with what is appropriate or not, especially if we are putting them up front to represent the Resonite Brand. Scantily clad or the "excessively-volumed" avatars tend to make people uncomfortable, but when gently confronted about it are met with explosive offense rather than discussion or compromise. From the point of view of newcomers/companies looking up Resonite.com and seeing that it might cause them to hesitate and walk away. ("If they dont like us then they can just leave" is the poisonous phrase I hear the most, as if it justifies our low population numbers and continued misbehaviors.)

If this concept was on its own thing, separate from the official Resonite website, then that is completely fine. Im more concerned with brand representation, if it is to be on the official site.
I have a more direct way of saying all I am concerned about but I am trying my hardest to be kind and censored, but all in all, I think this is a brand risk. Theres too much that can go wrong, if this were to be on the official Resonite website.

I really really want this platform to succeed. Froox made an amazing platform with infinite potential, but we *must* be careful about presentation so that anyone feels comfortable to join. I am saying all of this from a concerned point of view. The website as it is looks amazing and has genuinely been working to bring people in! Yes our community is weird, but think of the website like an interview - dont let them know you are weird until *after* hiring. So presentation is everything!
I probably wont be looking at replies to this. Im afraid Im going to get lynched from my opinion and dont want that stress. But above are my thoughts, at least. Thank you for coming to my ted talk. Sorry long post, I ate the potato.

**Reply by ProbablePrime** [reply]
Thank you for your feedback!

**Reply by ChromaShiba** [reply]
Seriously thank you @GearBell for posting on here!!!!!!!!!

The website upkeep factor is something I'm going to be heavily factoring into our approach now. Instead of having a constant opening for submissions, we can announce and limit submission periods to a select amount of time per year. 

Something like this absolutely needs to take into account the "Ego" factor when it comes to fostering healthy community communication. We don't want this to be another badge, but rather a resource for people to discover other creators and promote their own content. Like a marketplace without any exchange utility built into it directly (that's a whole can of worms at this stage haha).

Descriptions will have to undergo vetting on our side and adhere to our policies to be listed directly. For the direct listing format getting handled on our end, I'd plan on limiting that to a profile image and a single key art image, allowing the creator to provide a link to their preferred portfolio.  

Agreed on putting a presentable foot forward when it comes to specifically the key-art/profile images getting presented on our end. I'd like to be as transparent as possible to any submitting creator about what is tasteful (using our policies as a guiding light there) to showcase on this page. That work will be showcased next to other creators work, so being considerate about that content should be taken into account. At the end of the day, we might just reserve the right to veto, while being transparent as to our decision making process.

**Reply by TehTurk** [reply]
I appreciate the post Gearbell! I wasn't sure if we could actually comment on the brand or any of the risks involved because in the past that's been a bit hanky for community complaints/consideration. I wanted to bring that part up, but mainly wanted to focus on some path forward because in the past denial of a feature or someone's effort and work has always proved to be a sour/pain point. But regardless we are discussing and Chroma has also been super open to the discussion and consideration with alot of this too :)

**Reply by Banane9** [reply]
Very reasonable and thought out post, totally agree 👍

**Reply by brodokk** [reply]
Ok so this is a very interesting point of view.

This arguments could also been used for the events page that is worked on i feel...

Also Gearbell you was **really** near to use a domain that is a bit related to the subject but about some communities events and not showcasing them.

**Comment by Zyzyl** [comment]
The proposed webpages could be an interesting opportunity for community creators to advertise themselves and their creations. Gearbell already made some excellent points. I’d first like to echo the suggestion that any creator showcase and events calendar NOT be on the official resonite.com site. Secondly, I suggest that the Team minimise their involvement with the curation of content featured on that separate site.

I assume that the Resonite team wants Resonite to be used in business or academic organisations; as Gearbell mentioned, the proposed site could pose a serious risk to the Resonite brand in that context. The are Guidelines about the types of content acceptable on the platform. However, the amount of genuinely “safe for work” content (i.e. content which will not raise the eyebrows of potential corporate or academic institutions) will be substantially smaller than the whole set of content permitted on the public platform. As a simple illustrative case, I regularly see a public session open named “Cuntopia”; even if allowed on the platform, I wouldn’t advise allowing serious gendered slurs in content potential business partners may see when doing their due diligence. I am concerned the team will either end up in a constant battle with the community over what is and isn’t acceptable to show on the website, or risk alienating important business/academic interest. Unless the correct fine balance is struck, we may end up in a worst of both worlds situation.

To a significant extent, Resonite runs on goodwill – the Team should be extremely careful of creating situations which could degrade that due to Team/community conflict. Instead, consider handing over curation duties to community members with Moderation still retaining veto power. Perhaps this could work a bit like how the Mentors are ultimately overseen by the Moderation Leads, but they operate largely independently.
There are a few reasons I suggest this, mostly to do with community/Team relations:
- Whenever the Resonite Team makes a decision, it is likely to upset someone. This a natural consequence of having a diverse community. The curators of the proposed sites will probably end up taking a lot of flak for perceived bad decisions and may receive little praise for good ones. By delegating decisions on content curation, the Team will have to make fewer community-facing decisions. This would reduce the potential frequency of Team/community disagreement. Indeed, the Team may wish to take this approach with the current ‘Featured’ worlds function as well.
- Providing a platform to showcase creators’ work may have meaningful financial and reputational benefits for featured creators. This would incentivise trying to game the algorithm to increase a creator’s visibility. We repeatedly see this trend in similar contexts e.g. Search Engine Optimisation, clickbait, engagement farming… etc. If it is known that curation decisions are made by the Team, it is the Team’s decision-making process that users will be incentivised to hack. This could manifest as unwanted social interactions or attention directed at the Team. I understand there have been similar issues in the past with users trying to circumvent the standard bug reporting or prioritisation systems via social interaction with the Team.
- On a related note, I have seen several Team members express a wish that there were less of a clear divide between Team and community. If the team doesn’t wish to be seen as ‘special’, then they should avoid having more ‘special’ powers than necessary. In this context, the ability to affect creators’ financial or reputational success by having curation powers would classify as ‘special’.

Please let me know if any of that isn’t clear or needs expanding upon!
